src/js/game.js

```diff
@@ -0,0 +1,195 @@
+import { ERROR_MESSAGE, CONGRATS_MESSAGE, INPUT_TYPE } from "./constants.js";
+import { $ } from "./utils.js";
+
+export default function CarGame() {
+  const form = $("form");
+  const names = $("#names");
+  const namesInput = $("#names input");
+  const nameCards = $("#nameCards");
+  const nameSubmitBtn = names.querySelector("button");
+  const gameWinner = $("#winner");
+
+  let carNames = [];
+  let times = "";
+  let carsDistance = [];
+  let winner = [];
+
+  namesInput.addEventListener("keyup", ({ target }) => {
+    carNames = target.value;
+    carNames = checkCarNames(carNames);
+  });
+
+  function onCarNamesSubmit() {
+    nameSubmitBtn.addEventListener("click", () => {
+      showGameTimesInput();
+      onGameTimesSubmit();
+      preventInput(INPUT_TYPE.NAME);
+    });
+  }
+
+  function checkCarNames(names) {
+    const namesArr = names.split(",");
+    const nameBlankCheck = namesArr
+      .map(name => name.replace(/(^\s*)|(\s*$)/gi, ""))
+      .filter(name => !!name);
```

Comment:
별거 아니지만 Boolean 생성자를 전달해도 될것같네요 😃 
```suggestion
      .filter(Boolean);
```